src/test/java/com/epam/tc/hw5/steps/ActionStep.java

```diff
@@ -0,0 +1,82 @@
+package com.epam.tc.hw5.steps;
+
+import com.epam.tc.hw5.DifferentElementsPage;
+import com.epam.tc.hw5.DriverSetup;
+import com.epam.tc.hw5.MainPage;
+import com.epam.tc.hw5.UserTablePage;
+import com.epam.tc.hw5.fragments.HeaderMenu;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.When;
+
+public class ActionStep extends DriverSetup {
+
+    MainPage mainPage = new MainPage(driver);
+    DifferentElementsPage differentElementsPage = new DifferentElementsPage(driver);
+    HeaderMenu headerMenu  = new HeaderMenu(driver);
+    UserTablePage userTablePage = new UserTablePage(driver);
+
+    @Given("User opens website")
+    public void openWebSite() {
+        mainPage.openMainPage();
+    }
+
+    @Given("I open JDI GitHub site")
+    public void openJdiGitHubWebsite() {
+        mainPage.openMainPage();
+    }
```

Comment:
Please, choose "User" or "I" for definition of your steps :)

Reply:
Updated the name of all methods and their description